### PR TITLE
get-started: update .net deployment

### PR DESCRIPTION
**Pull request by craig-osterhout**
### Proposed changes

Update the Deployment section in the .NET guide to be like the Node.js guide.
 - Remove cloud integrations
 - Update the topic to use the sample app

https://deploy-preview-18400--docsdocker.netlify.app/language/dotnet/deploy/

### Related issues (optional)

ENGDOCS-1713

### CODE REVIEW

File: content/language/dotnet/deploy.md [hunk 1]
```diff
@@ -1,11 +1,212 @@
...
+## Overview
+
+In this section, you'll learn how to use Docker Desktop to deploy your
+application to a fully-featured Kubernetes environment on your development
+machine. This allows you to test and debug your workloads on Kubernetes locally
+before deploying.
+
+## Create a Kubernetes YAML file
+
+In your `docker-dotnet-sample` directory, create a file name
```
Comment (stephaurelio) [inline content/language/dotnet/deploy.md:23]: Possible typo: 
```suggestion
In your `docker-dotnet-sample` directory, create a file named
```

File: content/language/dotnet/deploy.md [hunk 2]
```diff
@@ -1,11 +1,212 @@
...
+    the image built by GitHub Actions in [Configure CI/CD for your .NET application](configure-ci-cd.md).
+ - A NodePort service, which will route traffic from port 30001 on your host to
+   port 8080 inside the pods it routes to, allowing you to reach your app
+   from the network.
+
+To learn more about Kubernetes objects, see the [Kubernetes documentation](https://kubernetes.io/docs/home/).
+
+## Deploy and check your application
+
+1. In a terminal, navigate the `docker-dotnet-sample` directory
```
Comment (stephaurelio) [inline content/language/dotnet/deploy.md:149]: Small typo: 
```suggestion
1. In a terminal, navigate to the `docker-dotnet-sample` directory
```

File: content/language/dotnet/deploy.md [hunk 3]
```diff
@@ -1,11 +1,212 @@
...
+  selector:
+    service: db
+status:
+  loadBalancer: {}
+```
+
+In this Kubernetes YAML file, there are four objects, separated by the `---`. In addition to a Service and Deployment for the database, the other two objects are:
+
+ - A Deployment, describing a scalable group of identical pods. In this case,
+   you'll get just one replica, or copy of your pod, and that pod (which is
```
Comment (stephaurelio) [inline content/language/dotnet/deploy.md:138]: Potential to break this up to make it easier to read: 
```suggestion
   you'll get just one replica, or copy of your pod. That pod (which is
```

**Review by stephaurelio — COMMENTED** [review]
Left a few small comments! Looks good ✨

File: content/language/dotnet/deploy.md [hunk 4]
```diff
@@ -135,9 +135,10 @@ status:
 In this Kubernetes YAML file, there are four objects, separated by the `---`. In addition to a Service and Deployment for the database, the other two objects are:
 
  - A Deployment, describing a scalable group of identical pods. In this case,
-   you'll get just one replica, or copy of your pod, and that pod (which is
-   described under the template: key) has just one container in it, based off of
-    the image built by GitHub Actions in [Configure CI/CD for your .NET application](configure-ci-cd.md).
+   you'll get just one replica, or copy of your pod. That pod, which is
+   described under `template`, has just one container in it. The container is
+    based off of the image built by GitHub Actions in [Configure CI/CD for your
```
Comment (dvdksn) [inline content/language/dotnet/deploy.md:140]: ?

```suggestion
    created from the image built by GitHub Actions in [Configure CI/CD for your
```

**Review by dvdksn — APPROVED** [review]
LGTM!

**Pull request merged**